policy/common/methods.go

```diff
@@ -79,11 +81,21 @@ func (m *Methods) Candidates(ctx context.Context, prctx pull.Context) ([]*Candid
 
 		for _, r := range reviews {
 			if r.State == m.GithubReviewState {
-				candidates = append(candidates, &Candidate{
-					User:      r.Author,
-					CreatedAt: r.CreatedAt,
-					UpdatedAt: r.UpdatedAt,
-				})
+				if len(m.GithubReviewComments) > 0 || len(m.GithubReviewCommentPatterns) > 0 {
```

Comment:
Because of the way method defaults work, I don't think this will work as expected (or at least as we discussed in the issue.) We only use the defaults if the `methods` block is completely absent. If the user sets any fields, no defaults are used. That means that with the current code, users will have to set both `github_reviews` and `github_review_comment_patterns` to have things work.

We may want to change the condition on L76 to `m.GitubReview || len(m.GithubReviewCommandPatterns) > 0` so that setting either field will enable review processing - unless we actually want to require that both fields are set.

Reply:
ahh ok, i think i misunderstood how the defaults were applied. Lemme look into fixing this.